src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {

```

Comment:
Think about which variables should be local in the method and which should be class-level. It looks like you've done this well by making 'random' and 'colorSupplier' class-level variables since they are used across multiple method calls.